live/sources/String+PrintWithColor.swift

```diff
@@ -29,6 +29,7 @@ extension String {
      *  - Parameter color: The color to print the string using
      */
     func print(withColor color: AnsiColor) {
-        Swift.print(color.rawValue + self)
+        /// Append a white color code to the string, to reset the console color
+        Swift.print(color.rawValue + self + AnsiColor.white.rawValue)
```

Comment:
How about using the “reset” ANSI value instead? Setting it to white might cause issues for user’s that use a white background color and black foreground. The “reset” value instead just clears any custom formatting. Returning the text that comes afterwards to the terminals default colors.

The ansi value is:
`[0m`

So we’d add a enum value like:
```swift
case reset = "\u{001B}[0m"
```